### DISCUSSION TITLE
Anyone considering taking over this project?

### DISCUSSION THREAD

**Discussion by bastaware** [bosskmk/pluto_grid#976] [General]
It would be a shame for such a good project to stall completely

**Comment by darkomenx** [comment]
Are you contacting the author of this project before ? What he said ?

**Comment by bastaware** [comment]
Yes. 6 month ago, he told me he was not feeling well, and seemed to have lost his motivation.

I've tried contacting him again, but this time I didn't get any answer.

See https://github.com/bosskmk/pluto_grid/issues/968

**Comment by darkomenx** [comment]
Ok ok I understand. Thank you for your lightning.
Do you have already try the new fork pluto_grid_plus ?

Perhaps I can help but alone I can't and without help to understand code and test I can't produce anything.

**Reply by doonfrs** [reply]
I am maintaining pluto_grid_plus, we depend  on it on multiple projects, so I will :
* keep it up to date with flutter / dark Sdk
* review and merge pull requests from the community 
* fix bugs

* Many bugs have already been fixed, and some new features have merged, you can check the commit log.
It is team work but needs someone to manage it.

https://pub.dev/packages/pluto_grid_plus
https://github.com/doonfrs/pluto_grid_plus

**Reply by darkomenx** [reply]
In my part, I also need to add just a little features into this package.
I can suggest change into a pull request but I don't understand how to work into a package and modify code that previously tested in my part (without I project that use it for testing change).

**Reply by doonfrs** [reply]
You can post them in the discussions section
https://github.com/doonfrs/pluto_grid_plus/discussions

**Comment by VittorioParagallo** [comment]
Hi all,
the pluto_grid_plus is discontinued as well in favour of https://github.com/doonfrs/trina_grid
just i saw that the owner of pluto_grid did a merge in march after long time, so he's alive, may be it could be good to talk to him to work on a unique project?
@bosskmk ?

**Reply by doonfrs** [reply]
This was expected—open-source maintainers often share their work out of passion, but may eventually move on, as it's unpaid time.
[Trina Grid](https://github.com/doonfrs/trina_grid) began as a fork, which I later rebranded and actively continued developing. It’s a reliable, actively maintained project that remains MIT licensed.

I’ve added clearer documentation, improved demo screens and code examples, and introduced many useful new features. You can find the [main documentation here](https://github.com/doonfrs/trina_grid/blob/main/doc/index.md).

To support migration, I also created a [Pluto-to-Trina guide](https://github.com/doonfrs/trina_grid/blob/main/doc/migration/pluto-to-trina.md) and a [migration tool](https://github.com/doonfrs/trina_grid/blob/main/doc/migration/migration-tool.md).

I welcome contributions to help improve the project and keep it sustainable without burning out.
